src/theory/arrays/theory_arrays.cpp

```diff
@@ -2247,10 +2247,8 @@ void TheoryArrays::conflict(TNode a, TNode b) {
 
     if (d_proofsEnabled) {
       proof->debug_print("pf::array");
-      proof_array = new ProofArray( proof );
-      proof_array->setRowMergeTag(d_reasonRow);
-      proof_array->setRow1MergeTag(d_reasonRow1);
-      proof_array->setExtMergeTag(d_reasonExt);
+      proof_array = new ProofArray(proof, /*row=*/d_reasonRow,
+                                   /*row1=*/d_reasonRow1, /*ext=*/d_reasonExt);
```

Comment:
Are these comments really required? Should be obvious from the interface of ProofArray?

Reply:
These specially formatted comments enable tools like clang-tidy to warn if there is a mismatch on the arguments. The danger is that someone has the order of the parameters wrong (or the order changes and a usage is not updated). The function has 3 unsigned parameters in a row and 2 of these differ by 1 character. Some extra paranoia and nudging to be careful seems justified IMO.

Reply:
OK. SGTM.